### DISCUSSION TITLE
Roadmap for ink stitch?

### DISCUSSION THREAD

**Discussion by joeynuggetz** [inkstitch/inkstitch#1602] [General]
Is there a roadmap of features that are planned? I really think the potential here is great news for digitizers. My wilcom hatch demo is about a week away from expiring and I’m sure I can accomplish 90% of what I need to do with inkstitch but the quality of life things in hatch are tempting me to purchase it. Could you guys fork Inkscape and build an open source digitizer and strip out the useless stuff so we can skip having to access the extensions menu :-) just kidding. Keep up the great work everyone.

**Comment by lexelby** [comment]
I'm glad you're excited. :)

I'm afraid there is no roadmap, but we do make steady, if slow, progress.  All of us seem to have a lot of other pulls on our time right now, but that changes over time.  Who knows what the future may bring?

We do have a lot of exciting features underway in the pull requests section, which you could try out if you're brave.  And of course, contributions are always welcome!

We do have at least one professional digitizer who said he uses Ink/Stitch, but we may not be ready for primetime just yet -- maybe that Wilcom purchase might be a good idea.  But if you do use Ink/Stitch, we'd love to hear what you think.
